edx_rest_framework_extensions/settings.py

```diff
@@ -11,6 +11,7 @@
 
 DEFAULT_SETTINGS = {
     'OAUTH2_USER_INFO_URL': None,
+    'JWT_PAYLOAD_ATTRIBUTES': ('email',)
```

Comment:
Consider a name that relates this to the fact that the values will be used to update the user model (e.g. `JWT_PAYLOAD_USER_ATTRIBUTES`).